Foundation/NSError.swift

```diff
@@ -147,7 +147,7 @@ open class NSError : NSObject, NSCopying, NSSecureCoding, NSCoding {
     internal typealias UserInfoProvider = (_ error: Error, _ key: String) -> Any?
     internal static var userInfoProviders = [String: UserInfoProvider]()
     
-    open class func setUserInfoValueProvider(forDomain errorDomain: String, provider: (/* @escaping */ (Error, String) -> Any?)?) {
+    open class func setUserInfoValueProvider(forDomain errorDomain: String, provider: (@escaping (Error, String) -> Any?)?) {
```

Comment:
This must drop the `@escaping` before it can build. Optional<X> where X is a closure type is always implicitly escaping.